indy_common/state/domain.py

```diff
@@ -135,6 +144,22 @@ def prepare_revoc_def_for_state(txn):
     return path, value_bytes
 
 
+def prepare_revoc_reg_entry_for_state(txn):
+    author_did = txn.get(f.IDENTIFIER.nm)
+    revoc_reg_def_id = txn.get(REVOC_REG_DEF_ID)
+    assert author_did
+    assert revoc_reg_def_id
+    path = make_state_path_for_revoc_reg_entry(authors_did=author_did,
+                                               revoc_reg_def_id=revoc_reg_def_id)
+
+    seq_no = txn[f.SEQ_NO.nm]
+    txn_time = txn[TXN_TIME]
+    assert seq_no
+    assert txn_time
+    value_bytes = encode_state_value(txn, seq_no, txn_time)
```

Comment:
The state should accumulate all issued and revoked indices.
Please also add a test for this.